CHANGELOG.next.asciidoc

```diff
@@ -192,6 +193,42 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `providers` setting to `add_cloud_metadata` processor. {pull}13812[13812]
 - Ensure that init containers are no longer tailed after they stop {pull}14394[14394]
 - Fingerprint processor adds a new xxhash hashing algorithm {pull}15418[15418]
+- Add document_id setting to decode_json_fields processor. {pull}15859[15859]
```

Comment:
Hey @ycombinator ! Is all this normal however?

Reply:
Nope, it's definitely not! 😬 Not sure what happened here, fixing.

Reply:
Okay, I think I have this finally in the right state. What you see here should be a backport of https://github.com/elastic/beats/pull/17958/files and https://github.com/elastic/beats/pull/18233/files.